### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by onmyway133**
- [x] Update README
- [x] Update banner image
- [x] Update icon

### CODE REVIEW

**Comment by zenangst** [comment]
Might want to work a bit on the icon, it feels like it doesn't really fit in with the rest of the family.

File: README.md [hunk 1]
```diff
@@ -9,14 +11,99 @@
...
+
+```swift
+Config.Permission.image = UIImage(named: ImageList.Gallery.cameraIcon)
+Config.Font.Text.bold = UIFont(name: FontList.OpenSans.bold, size: 14)!
+Config.Camera.recordLocation = true
+```
+
+### Video Editor
+
+`Galery` cares more about video with its editing functionalities. We have `VideoEditor` and `AdvancedVideoEditor` to trim, resize, scale and define quality of the selected video
```
Comment (zenangst) [inline README.md]: typo: `Galery` -> `Gallery`

Comment (zenangst) [inline README.md]: I would change `functionalities` to `capabilities`

**Comment by zenangst** [comment]
I would change the banner image to show: `Photos` <-> `Camera` <-> `Video`

**Comment by zenangst** [comment]
@onmyway133 nice README :)

**Comment by onmyway133** [comment]
@zenangst credit goes to our awesome designer ❤️

**Comment by zenangst** [comment]
the banner looks much better now 😎

**Pull request merged**